sdk/tables/aztables/tableClient.go

```diff
@@ -7,34 +7,69 @@ import (
 	"context"
```

Comment:
FYI since this is for the UX study about how Go-ish our SDKs are 😝...the file names need to be fixed to be snake cased